app/src/main/java/com/orange/ouds/app/ui/tokens/Token.kt

```diff
@@ -19,14 +19,17 @@ import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.unit.Dp
 import com.orange.ouds.app.R
 
-data class Token<T>(val name: String, val value: T) {
+data class Token<T>(val name: String, val value: @Composable () -> T) {
     val literalValue: String
         @Composable
-        get() = when (value) {
-            is Color -> value.toHexString()
-            is Float -> stringResource(id = R.string.app_tokens_floatFormat_label, value)
-            is Dp -> stringResource(id = R.string.app_tokens_dpFormat_label, value.toString().replace(".0.dp", "").substringBeforeLast(".dp"))
-            is TextStyle -> stringResource(id = R.string.app_tokens_spFormat_label, value.fontSize.toString().replace(".0.sp", "").substringBeforeLast(".sp"))
+        get() = when (val v = value()) {
+            is Color -> v.toHexString()
+            is Float -> stringResource(id = R.string.app_tokens_floatFormat_label, v)
+            is Dp -> stringResource(id = R.string.app_tokens_dpFormat_label, v.toString().replace(".0.dp", "").substringBeforeLast(".dp"))
+            is TextStyle -> stringResource(
+                id = R.string.app_tokens_spFormat_label,
+                v.fontSize.toString().replace(".0.sp", "").substringBeforeLast(".sp")
+            )
             else -> value.toString()
```

Comment:
This should be `v` instead of `value` because this will print a string representation of the composable method.

By the way, we could rename `v` to `value`, there is no ambiguity from the compiler point of view if I'm not wrong.